src/emulator/modules/SceGxm/src/SceGxm.cpp

```diff
@@ -57,7 +58,7 @@ static void before_callback(const FunctionCall &fn) {
 static void after_callback(const FunctionCall &fn) {
     MICROPROFILE_LEAVE();
     for (GLenum error = glGetError(); error != GL_NO_ERROR; error = glGetError()) {
-        std::cerr << "OpenGL: " << fn.function->name() << " set error " << error << "." << std::endl;
+        LOG_ERROR("OpenGL: {} set error {}.", fn.function->name(), static_cast<unsigned int>(error));
```

Comment:
`glbinding` does some kind of magic to convert OpenGL errors to strings here. Is that preserved? The cast to `unsigned int` suggests it's not.

Reply:
Ohh I didn't know that. Guess I'll have to make a stringstream and feed it that way then, cool.